Holovibes/sources/cli.cc

```diff
@@ -235,7 +251,7 @@ static void main_loop(holovibes::Holovibes& holovibes)
             else
             {
                 // Change the speed of the progress bar according to the nb of frames skip
-                progress_bar(progress->first * (holovibes::api::get_nb_frame_skip() + 1), progress->second, 40);
+                progress_bar(progress->first, progress->second, 40);
```

Comment:
Merge conflict ?

Reply:
The calculation inside the `cli` for the progress bar recording where wrong. The frame skip is already calculated inside the `RecordWorker` so you just need to get how many frames were recorded